pennylane_cirq/qsim_device.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=too-many-arguments
+"""
+This module provides the ``QSimDevice`` and ``QSimhDevice`` from Cirq.
+"""
+import cirq
+import numpy as np
+
+try:
+    import qsimcirq
+except ImportError:
+    raise ImportError(
+        "QSim Cirq is needed for the QSim devices to work."
+        "\nIt can be installed using pip:"
+        "\n\npip install qsimcirq"
+    )
```

Comment:
Are we avoiding making this a dependency as `qsimcirq` might be difficult to install?

Reply:
Yes, I think that was the case. I didn't have any issues myself, but I believe @josh146 mentioned something about this being potentially difficult to build.